### DISCUSSION TITLE
PyGMTInSAR

### DISCUSSION THREAD

**Discussion by Abb16** [gmtsar/gmtsar#845] [General]
I want to learn from the scratch about the PyGMTInSAR would you mind to help me please? What shlould am I to do to learn fast and  detail. Thank you In advanced!

**Comment by Xiaohua-Eric-Xu** [comment]
This is likely a user adopted version (https://github.com/mobigroup/gmtsar). Please get to that repository and pose your questions/problems/discussions there, as due to a small developer team, we can only be responsible for the development of GMTSAR and help its users. 

Also, FYI, though we encourage users to take GMTSAR and adopt it for building new processing chains or InSAR approaches, we could not guarantee on quality of their product. You may need to contact the authors of the user adopted repositories for any question or problems.
